### PR TITLE
Feature/team ui

### PR DESCRIPTION
**Pull request by turso**
- Queryparams now are flushed when moving away from series view
- Complete basic layout shift so elements can vertically pass the whole field
- All elements are more in the field of view of the user
- Team card layout to match Figma sketches
- Moved files to right places with right names
- 1024 px breakpoint to all major layouts

### CODE REVIEW

File: end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot [hunk 1]
```diff
@@ -27,29 +27,29 @@ Test Navigation to Last Builds
...
       Append To List    ${series_names}    ${header_text}
   END
 
 Test Overview of All Stored Series
     Wait Until Element is Enabled   ${series_list}
-    
+
     FOR    ${header}    IN    @{series_names}
-      Click Element     //h3[.="${header}"]/ancestor-or-self::div[2]/div[2]/div[1]
+      Click Element     //h4[.="${header}"]/ancestor-or-self::section[2]/div[2]/div[1]
```
Comment (vade5g) [inline end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot]: Ehmm, this selector is not a good idea, we could have a chat about these together.

File: end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot [hunk 2]
```diff
@@ -27,29 +27,29 @@ Test Navigation to Last Builds
...
+      Go Back
       Wait Until Element is Enabled   ${series_list}
     END
 
 
 Test Last Builds of All Stored Series
     Wait Until Element is Enabled   ${series_list}
     FOR    ${header}    IN    @{series_names}
-      Click Element     //h3[.="${header}"]/ancestor-or-self::div[2]/div[2]/div[2]
+      Click Element     //h4[.="${header}"]/ancestor-or-self::section[2]/div[2]/div[2]
```
Comment (vade5g) [inline end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot]: Same goes here.

File: end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot [hunk 3]
```diff
@@ -27,29 +27,29 @@ Test Navigation to Last Builds
...
       Append To List    ${series_names}    ${header_text}
   END
 
 Test Overview of All Stored Series
     Wait Until Element is Enabled   ${series_list}
-    
+
     FOR    ${header}    IN    @{series_names}
-      Click Element     //h3[.="${header}"]/ancestor-or-self::div[2]/div[2]/div[1]
+      Click Element     //h4[.="${header}"]/ancestor-or-self::section[1]/div[1]/div[1]
```
Comment (vade5g) [inline end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot:41]: We can surely get a better locator for this element. Let's discuss this together.

**Review by lakooi — APPROVED** [review]

File: end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot (hunk 1 above)
Comment (lakooi) [reply end_to_end_tests/robot_tests/frontend/series_page_tests/navigation.robot]: I will change these to function like so
-> The cards will have dynamic ids for different parts based on the unique series name, such as ci_frontend_latest
-> We can then use unique series name as a locator with a the id

Currently What the locator does is it finds the correct header element based on content and then moves one container up from the element after which the element is then found.

File: end_to_end_tests/robot_tests/frontend/team_page_tests/navigation.robot [hunk 4]
```diff
@@ -29,8 +29,8 @@ Test Team Navigation using Team Headers
   Wait Until Element is Enabled   ${teams_xpath}
   #This long identifier points to the header part of the component that matches the header text
   FOR    ${header}    IN    @{team_names}
-      Click Element     //*[@id="team"]/div/div[*]/h3[contains(text(),"${header}")]
+      Click Element     //*[@id="team"]/div/section[*]/h4[contains(text(),"${header}")]
```
Comment (vade5g) [inline end_to_end_tests/robot_tests/frontend/team_page_tests/navigation.robot:32]: ☝️

File: frontend/src/contexts/reducer.js [hunk 5]
```diff
@@ -159,6 +159,12 @@ const reducer = (state, action) => {
                 ...state,
                 selectedSuiteState: null,
             };
+        case 'flushParams':
```
Comment (vade5g) [inline frontend/src/contexts/reducer.js:162]: Just a comment, flushParams could be either used to remove all params, or named more accordingly if it's not mean't to flush all the Parameters there are.

**Review by vade5g — APPROVED** [review]

File: frontend/src/contexts/reducer.js (hunk 5 above)
Comment (turso) [reply frontend/src/contexts/reducer.js:162]: I can change this to flushQueryParams

**Pull request merged**